src/browser/ui/dom/SVGDOMPropertyConfig.js

```diff
@@ -83,7 +114,18 @@ var SVGDOMPropertyConfig = {
     strokeOpacity: 'stroke-opacity',
     strokeWidth: 'stroke-width',
     textAnchor: 'text-anchor',
-    viewBox: 'viewBox'
+    viewBox: 'viewBox',
+    xlinkActuate: 'xlink:actuate',
```

Comment:
If I'm understanding right, the name that you pass to setAttributeNS should actually be just 'actuate' here. When generating the markup, we do want to write xlink:actuate though…